### DISCUSSION TITLE
https://angular.love/under-the-hood-of-web-bundlers-e-g-webpack

### DISCUSSION THREAD

**Discussion by giscus** [HouseOfAngular/angular-love#331] [General]
# https://angular.love/under-the-hood-of-web-bundlers-e-g-webpack

Angular.love - a place for all Angular enthusiasts created to inspire and educate.

https://angular.love/under-the-hood-of-web-bundlers-e-g-webpack

**Comment by dangnguyen1004** [comment]
F*cking awesome
